core/src/main/java/com/expedia/www/haystack/client/dispatchers/clients/LoggerClient.java

```diff
@@ -0,0 +1,75 @@
+package com.expedia.www.haystack.client.dispatchers.clients;
+
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.expedia.www.haystack.client.Span;
+import com.expedia.www.haystack.client.dispatchers.formats.Format;
+
+public class LoggerClient implements Client {
+    private static final Logger LOGGER = LoggerFactory.getLogger(LoggerClient.class);
+
+    private Format<?> format;
+    private Logger logger;
```

Comment:
Can you please add a comment about why there are two loggers, one static and one an instance variable?

Reply:
I can add a comment; but it's for the normal reasons you have a `static` Logger (collecting logging about the class operations).  The other one is for the actual client data... ie the instance data.